Chess-FromZeroToHero.Contracts/Dtos/User/UserWithIdDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Chess_FromZeroToHero.Contracts.Dtos.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chess_FromZeroToHero.Contracts.Dtos.User
+{
+    public class UserWithIdDto : BaseWithIdDto
+    {
+        public string Username { get; set; }
+
+        public string Password { get; set; }
+
+        public int Rating { get; set; }
+
+        public string Name { get; set; }
+
+        public int Age { get; set; }
```

Comment:
It's more flexible if you store the birth date instead of the age, because it will be easier to work with once the person ages.